### PR TITLE
Intent cleaned responses

### PR DESCRIPTION
**Pull request by alidaniell**
Addressing Issue #108, Alexa script audit and clean up.
Modified these files to reduce syllables and retain clarity.

mycity/mycity/intents/get_alerts_intent.py
mycity/mycity/intents/snow_parking_intent.py
mycity/mycity/intents/trash_intent.py
mycity/mycity/mycity_controller.py

Some snow emergency parking responses are still wordy, because the comments are pulled from the Comments column of http://bostonopendata-boston.opendata.arcgis.com/datasets/53ebc23fcc654111b642f70e61c63852_0.csv. How to deal with the wordy Comments column is not a matter for this bug but is still an issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -206,7 +206,23 @@ containing directory instead of its contents, this code is not available.
...
+Boston's trash day schedule here: https://www.boston.gov/trash-day-schedule
+labelled 'Powered by ReCollect'.
+
+trash_intent.py accesses the information at recollect.net
+
+jmartini noted on 02/13/2018 in issue #39: "The Boston digital team stated that the trash day info at data.boston.gov is often out of date. They are currently using a 3rd party called ReCollect to manage the newest trash day info."
+
 ### https://data.boston.gov
+
+**DEPRECATED**
```
Comment (jmartini) [inline README.md]: I'm not sure we should call this whole site deprecated. It looks like there is still some valid data on there. It's unfortantely a case by case basis.

File: mycity/mycity/intents/get_alerts_intent.py [hunk 2]
```diff
@@ -82,7 +82,7 @@ def alerts_to_speech_output(alerts):
     for alert in alerts.values():
         all_alerts += alert + ' '
     if all_alerts.strip() == "":        # this is a kludgy fix for the {'alert header': ''} bug 
-        return "There are no alerts. City services are operating on their normal schedule."       
+        return "I see no alerts. City services are running on their normal schedules."
```
Comment (jmartini) [inline mycity/mycity/intents/get_alerts_intent.py]: "I see no alerts." sounds like Alexa is not very confident versus saying "There are no alerts" or similar.

File: mycity/mycity/intents/snow_parking_intent.py [hunk 3]
```diff
@@ -16,17 +16,17 @@
 DRIVING_DIST = g_maps_utils.DRIVING_DISTANCE_TEXT_KEY
 DRIVING_TIME = g_maps_utils.DRIVING_TIME_TEXT_KEY
 OUTPUT_SPEECH_FORMAT = \
-    ("The closest snow emergency parking location, {Name}, is at "
+    ("The closest snow emergency parking lot, {Name}, is at "
      "{Address}. It is {" + DRIVING_DIST + "} away and should take "
-     "you {" + DRIVING_TIME + "} to drive there. The parking lot has "
+     "you {" + DRIVING_TIME + "} to drive there from home. The lot has "
```
Comment (jmartini) [inline mycity/mycity/intents/snow_parking_intent.py]: Do we know the user is at home? What if they are at the office?

File: mycity/mycity/intents/get_alerts_intent.py (hunk 2 above)
Comment (alidaniell) [reply mycity/mycity/intents/get_alerts_intent.py]: Your point makes sense  - I was trying to make Alexa sound more friendly and human.

File: mycity/mycity/intents/snow_parking_intent.py (hunk 3 above)
Comment (alidaniell) [reply mycity/mycity/intents/snow_parking_intent.py]: Excellent point - I didn't consider they could have given an address that was not where they lived.  Let's delete the "from home".

File: README.md (hunk 1 above)
Comment (alidaniell) [reply README.md]: Yes, let's get rid of that **DEPERECATED**

**Pull request merged**